src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -1204,6 +1204,98 @@
         az acr webhook update -n MyWebhook -r MyRegistry --status disabled
 """
 
+# region connected-acr TODO
+helps['acr connected-acr'] = """
+type: group
+short-summary: Manage Edge/On-Prem Container Registries with Azure Container Registries.
+"""
+
+helps['acr connected-acr create'] = """
+type: command
+short-summary: Create a connected registry for an Azure Container Registry.
```

Comment:
If the parent is an ACR, the user should not need to give us the login server. The ACR name is provided in the required `--registry` parameter as this is a proxy resource.